codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -26,6 +26,12 @@ customising->customizing
 dialogue->dialog
 digitise->digitize
 digitising->digitizing
+discretisation->discretization
+discretise->discretize
+discretised->discretized
```

Comment:
It seems these particular British words are too obscure for our current Aspell dictionary. #1485 might resolve that issue, but for now you should probably just remove these ones.

Reply:
I can't find discretise in any variant of the word list: http://app.aspell.net/lookup?dict=en_GB-large&words=discretise